bluetooth/src/androidLibMain/kotlin/device/DeviceWrapper.kt

```diff
@@ -60,7 +62,7 @@ class DefaultDeviceWrapper(override val device: BluetoothDevice) : DeviceWrapper
 
     override fun createBond() {
         try {
-            device.javaClass.getMethod("createBond").invoke(device)
+            device.createBond()
```

Comment:
Maybe removeBond also now available to change in method above?

Reply:
No it isnt, I checked :)